[Test] Re-Enable Release Stack Encrypted Log Connected Test

ParaskP7 opened the pull request:
This connected test was temporarily disabled as it was only failing in CircleCI. However, since CircleCI is not longer being used for this project, and instead, this project is now using Buildkite, this test is being re-enabled.

FYI: This work is based on my comment [here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2636#pullrequestreview-1249917384).

PS: Those test were disabled as part of [this PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1662).

-----

### To test

1. There is nothing much to test here.
2. Verifying that all the CI checks are successful should be enough.

Having said the above, note the fact that for some time now the `Connected Tests` job is not required for `FluxC` PRs to be merged. This is because we have some tests still failing there that need to be fixed before we can re-enabled that check. Thus, we will need to check if that specific `ReleaseStack_EncryptedLogTest` test passes in Firebase manually.

ParaskP7 committed:
Test: Re-enable release stack encrypted log test

This connected test was temporarily disabled as it was only failing in
CircleCI. However, since CircleCI is not longer being used for this
project, and instead, this project is now using Buildkite, this test
is being re-enabled.

ParaskP7 commented:
FYI: Actually, I am still seeing 1/2 test on this `ReleaseStack_EncryptedLogTest` suite failing, the `testQueueForUploadForInvalidUuid` (see test results [here](https://console.firebase.google.com/project/api-project-108380595987/testlab/histories/bh.f08c7ad6c1a268e8/matrices/6632754086758371012/executions/bs.c435089ec0a2124c/test-cases) and [here](https://console.firebase.google.com/project/api-project-108380595987/testlab/histories/bh.f08c7ad6c1a268e8/matrices/8927845962297825105/executions/bs.62c5328ea034fc69/test-cases)). Thus, I am not sure if that test suite is ready after all... 🤔

PS: The other [testQueueForUpload](https://console.firebase.google.com/project/api-project-108380595987/testlab/histories/bh.f08c7ad6c1a268e8/matrices/6632754086758371012/executions/bs.c435089ec0a2124c/testcases/325/test-cases) test seems to be working as expected. 🤷

malinajirka reviewed (approved):
Thanks for looking into this @ParaskP7! These things are easy to forget about and I truly appreciate you proactively fixing these types of 'rests'. 

Unfortunately, I'm not sure why the test is failing and my lack of knowledge+experience with encrypted logging doesn't help. As we discussed I'd suggest putting `@ignore` on the failing test so we improve the state a bit for now. Later, we can discuss it with someone who has experience with the logs as they might be able to figure out what's going on faster than I'd.

ParaskP7 commented:
I agree @malinajirka and thanks so much for reviewing this change, unfortunately it wasn't that straightforward... 😊

As suggested, I'll proceed with ignoring this one failing test and merge the PR afterwards. 👍

ParaskP7 committed:
Test: Ignore test queue for upload for invalid uuid

While 'testQueueForUpload' passes, 'testQueueForUploadForInvalidUuid'
fails and thus temporarily ignored.

ParaskP7 commented:
FYI: I did the change, but now tests are failing because of an unrelated `testLoadImageFromHTTPAuthSite` test failure. 🤷 

PS: I'll proceed with merging this anyway as I don't think this is related. We also know that connected tests are not stable anyway, thus that shouldn't block us, especially since the failing test failing is not related to the `ReleaseStack_EncryptedLogTest` test suite.
